docs/components/modeler/desktop-modeler/index.md

```diff
@@ -2,12 +2,12 @@
 id: index
 title: Desktop Modeler
 sidebar_label: About
-description: "Camunda Desktop Modeler is a desktop app for modeling BPMN, DMN, and Forms, compatible with Camunda 7 and Camunda 8."
+description: "Camunda Desktop Modeler is a companion desktop application for modeling BPMN, DMN, and Forms, compatible with Camunda 7 and Camunda 8. Designed to integrate seamlessly into professional software development workflows, it is intended to be used alongside your preferred IDE—not as a standalone tool."
```

Comment:
```suggestion
description: "Camunda Desktop Modeler is a desktop application for modeling BPMN, DMN, and Forms, compatible with Camunda 7 and Camunda 8. As a companion tool to your favorite IDE it is designed to integrate seamlessly into your professional software development environment."
```

I'd like to prevent the framing of "Don't us it standalone", but instead highlight the "better together (with an IDE, for implementing Camunda)" story.

What do you think @christinaausley?

Reply:
I'd also consider to align both this description and the introduction of the page (README).